helm/circles-infra-suite/templates/configmap.yaml

```diff
@@ -42,6 +42,10 @@ data:
   # ~~~~~~~~~~~~~~~~~~~~
   DATABASE_SOURCE: graph
 
+  # Api timeout Configuration (in ms)
+  # ~~~~~~~~~~~~~~~~~~~~
+  TRANSFER_STEPS_TIMEOUT: 150000
```

Comment:
Maybe it's a good idea to add it to values-production.yml

Reply:
Is that because we want different values for prod and stg?

Reply:
more for consistency - maybe even in values.yml? In configmap we define the variables and in values.yml its actual value

Reply:
good! But I didn't manage to make it work with the values files